src/request_collector/server.py

```diff
@@ -63,9 +63,10 @@ def _http_retry_delay() -> Iterable[float]:
     )
     login_or_register(client, signer=LocalSigner(private_key=decode_hex(private_key)))
 
+    room_name = '_'.join(['raiden', str(chain_id), MONITORING_BROADCASTING_ROOM])
```

Comment:
I'm implementing https://github.com/raiden-network/raiden/issues/3469

Reply:
Yeah, this included some other stuff to make the chain_id available.

Reply:
`chain_id` is not enough, you need also chain name, as per `raiden_contracts.constants`. https://github.com/raiden-network/raiden/pull/3471 takes care of everything